src/components/EndScreen.js

```diff
@@ -0,0 +1,12 @@
+function EndScreen({ counter }) {
+    return (
+        <div>
+            <div>{counter}</div>
+            <div className='success'>Gut gemacht!</div>
+            <button className='whatsapp-button'>Nochmal spielen</button>
```

Comment:
hier naming ändern, weil der button nichts mehr mit WA zu tun hat (auch nicht vergessen in css zu ändern ;))